docs/components/connectors/out-of-the-box-connectors/kafka-inbound.md

```diff
@@ -0,0 +1,143 @@
+---
+id: kafka-inbound
+title: Kafka Consumer Connector
+sidebar_label: Kafka Consumer Connector
+description: "The Kafka Consumer Connector allows you to connect your BPMN service with Kafka. Learn how to create a Kafka Consumer Connector and make it executable."
+---
+
+The **Kafka Consumer Connector** allows you to connect your BPMN service with [Kafka](https://kafka.apache.org/).
+
+## Prerequisites
+
+To use the **Kafka Consumer Connector**, you need to have a Kafka instance with configured bootstrap server.
+It is highly recommended not to expose your sensitive data as plain text, but rather use Camunda secrets. Follow our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
+
+## Create a Kafka Consumer Connector task
+
+1. Start building your BPMN diagram with a **Start Event** building block.
+2. Change its template to a Kafka Consumer.
+3. Fill in all required properties.
```

Comment:
@christinaausley Again, this is not Kafka specific. Should we extract a general "How to add an inbound connector to a process?" section?

Reply:
>  I always say it is better to overdocument and reference additional content where necessary, but if we aren't receiving much confusion here, we can always wait on it and keep an eye on usage.

Ok, maybe we need to have some kind of reusable template that we can include as it will be always the same for every inbound connector.